.scalafmt.conf

```diff
@@ -0,0 +1,34 @@
+version = "3.5.1"
```

Comment:
For the ease of reviewing, could we split Scalafmt into another PR?

Reply:
I will remove the scalafmt. I just needed it for formatting some of the files, but not all esp. AssemblyOption and AssemblyKeys.

@eed3si9n I kind of forgot, but when updating PRs, I should just do a new commit, right? Is it you who will take care of the squashing during merge?

Reply:
I am not too fussy about git history on plugin projects, so I'd probably keep the git history as-is if it makes sense to do so.

Reply:
if you want to stack up fixes on top, I can squash them on merge too.